app/docker/nginx/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx:stable-alpine
+
+COPY ./app/docker/nginx/site.conf /etc/nginx/conf.d/default.conf
```

Comment:
I understand it helps you, but benchmark kit for REST API is on the way and replace it. I will delete if after the merge.

Reply:
Great to hear you're going to release a dev kit! Hope it is simple to use. Another things that's bugging me is that repositories have the same name as the original project, e.g. symlex or symfony. That way developers typically have to rename it to avoid naming conflicts. A prefix like "benchmark-" or "phpbenchmarks-" would be amazing.

Reply:
I mean the GitHub repo name: When I fork `phpbenchmarks/symlex`, it is automatically named `lastzero/symlex` (moved the real symlex repo to `symlex/symlex`, but it all shows up as just `symlex` in PhpStorm which is confusing).